lsp-scala.el

```diff
@@ -60,9 +65,17 @@
   (lsp-send-execute-command "source-scan" ())
   )
 
+(eval-after-load 'lsp
```

Comment:
`with-eval-after-load` is improved version of `eval-after-load ` that doesn't require body to be quoted

Reply:
:+1: